### PR TITLE
Resolves ticket 2989 (previously CHEF-4446): Version numbers are now dis...

### PR DESCRIPTION
**Pull request by kungfujoe**
...played when synching cookbooks.

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: lib/chef/cookbook/synchronizer.rb [hunk 1]
```diff
@@ -131,7 +131,7 @@ def mark_file_synced(file)
       files_remaining_by_cookbook[file.cookbook] -= 1
 
       if files_remaining_by_cookbook[file.cookbook] == 0
-        @events.synchronized_cookbook(file.cookbook.name)
+        @events.synchronized_cookbook(file.cookbook.name, file.cookbook.version)
```
Comment (stevendanna) [inline lib/chef/cookbook/synchronizer.rb]: Are we concerned about breaking 3rd party formatters here?  If they implemented the old single-argument method, they will get an ArgumentError here.

**Comment by kungfujoe** [comment]
@stevendanna That's a great catch, and a great point. I've committed an update to the code to make it backwards compatible. I tested this by uploading the 4 changed files to my Chef Client, and ran it (and it worked as expected), and then I reverted doc.rb back to normal, and the code still ran without complaint (and without version numbers).

**Comment by smurawski** [comment]
@kungfujoe Can you sign the CLA?  That'll help us get this PR merged.

**Comment by juliandunn** [comment]
PR #3700 superseded this one.

**Pull request closed without merging**